src/core/__tests__/ReactMultiChildText-test.js

```diff
@@ -16,10 +16,13 @@
 require('mock-modules');
 
 var React = require('React');
+var ReactFragment = require('ReactFragment');
 var ReactTestUtils = require('ReactTestUtils');
 
 var reactComponentExpect = require('reactComponentExpect');
 
+var frag = ReactFragment.create;
```

Comment:
Other tests require in beforeEach, so we're already inconsistent in that regard.